cardano-config/src/Cardano/Tracing/ToObjectOrphans.hs

```diff
@@ -423,6 +423,7 @@ instance HasSeverityAnnotation (WithMuxBearer peer MuxTrace) where
     MuxTraceHandshakeServerError {} -> Error
     MuxTraceRecvDeltaQObservation {} -> Debug
     MuxTraceRecvDeltaQSample {} -> Info
+    MuxTraceSDUReadTimeoutException {} -> Notice
```

Comment:
Was this one missing? Shouldn't that have been a warning caught with `-Werror`?

Reply:
yes, that's why I added it.

Reply:
So that means `-Werror` is not enabled on CI :flushed: